autoload/neomake.vim

```diff
@@ -531,6 +536,7 @@ function! s:Make(options) abort
         let s:make_id += 1
         let s:make_options[s:make_id] = {
                     \ 'cwd': getcwd(),
+                    \ 'verbosity': get(g:, 'neomake_verbose', 1) + &verbose,
```

Comment:
> The default is 1 currently, but that's not set in stone, and logging levels should be reviewed anyway - currently I'm just stuffing most of it into the debug level (3).

This is what I mean.  If `&verbose` is `0`, then just use `g:neomake_verbose` since its default is `1`.

```vim
let verbosity = &verbose ? &verbose : get(g:, 'neomake_verbose', 1)
```